tidb-cloud/recovery-group-failover.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Failover and Reprotect Databases
+summary: Learn how to use a Recovery Group to Failover and Reprotect databases between TiDB Cloud clusters
+---
+
+# Failover and Reprotect Databases
+
+Databases that are part of a replication group are replicated from one cluster to another (typucally in a different region of the cloud service provider).
```

Comment:
typo: typucally

Reply:
Addressed in https://github.com/pingcap/docs/pull/17425/commits/da21f7502dd8af02816368bae7e09f501b693fa1